### PR TITLE
fix XAxis label rendering issue if data length is an even number or 1

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

File: lib/javascripts/Charts/XAxis.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
 import d3 from 'd3';
 import Axis from './Axis';
 import getDate from '../Helpers/getDate';
+import { isEven } from '../Helpers/number';
+
+
```
Comment (catherine-jones) [inline lib/javascripts/Charts/XAxis.js]: can we get rid of the double line?

File: lib/javascripts/Helpers/number.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const isEven = (n) => n % 2 == 0;
+
```
Comment (catherine-jones) [inline lib/javascripts/Helpers/number.js]: also should we get rid of this line as well?

**Comment by catherine-jones** [comment]
Maybe a couple of formatting issues that could be address but otherwise LGTM

File: lib/javascripts/Helpers/number.js (hunk 2 above)
Comment (elisechant) [inline lib/javascripts/Helpers/number.js]: this line should exist

**Pull request merged**